### ISSUE TITLE
feature: Add a tag for restricted communities

### ISSUE THREAD

**Issue by jcdeoferio** [username-is-required/reddark#45]
Subreddits like /r/Pics is kept as read-only but not private. It would be better to still highlight those subs as "private"

**Comment by jcdeoferio** [comment]
From https://old.reddit.com/r/pics.json

I think the flag you need to check is `data.children[0].subreddit_type`

**Comment by username-is-required** [comment]
right, this should hopefully be ready to push now. unfortunately reddit has been having issues thst made issues my app was having *worse*, to the point where mine’s now been effectively useless for probably more than an hour at this point. so i’ll push it when i can (hopefully today)
